website/docs/sdk-reference/dart.md

```diff
@@ -207,12 +268,54 @@ client.forceRefresh();
 ```
 > `getValue()` returns `defaultValue` if the cache is empty. Call `forceRefresh()` to update the cache.
 
+## Hooks
+
+With the following hooks you can subscribe particular events sent by the SDK:
```

Comment:
- subscribe => subscribe to ("to" is missing)
- send => fired (a suggestion only)

Reply:
✅